### PR TITLE
docs: add djamaile blog to site showcase

### PR DESCRIPTION
**Pull request by djamaile**
## Motivation

Made my personal blog/cv with docusaurus, thought it would be nice to showcase how I did it. 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

You can find my blog on https://djamaile.dev and the repo at https://github.com/djamaile/portfolio

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -383,6 +383,14 @@ const Users: User[] = [
     source: 'https://github.com/loft-sh/devspace',
     tags: ['opensource'],
   },
+  {
+    title: 'Djamaile Rahamat',
+    description: 'Djamaile Rahamat Blog on making cool stuff',
+    preview: require('./showcase/djamaile.png'),
+    website: 'https://djamail.dev/',
```
Comment (slorber) [inline website/src/data/users.tsx:390]: This url does not seem to be online.
We only add to showcase sites that are online :)

**Review by slorber — CHANGES_REQUESTED** [review]
Please ensure site is online. Netlify/Vercel are good simple hosting solutions

File: website/src/data/users.tsx (hunk 1 above)
Comment (djamaile) [reply website/src/data/users.tsx]: It is online, was a silly spelling mistake. My bad!

**Comment by slorber** [comment]
Thanks 👍

**Pull request merged**